Coverage build failure due to missing coverage for `library/std/src/thread/local.rs`

Generating coverage report for project Oak fails with the following error: 
```
error: /out/rustc/6d395a1c2946c79966490f5b1e6e619d3a713e6b/library/std/src/thread/local.rs: No such file or directory
warning: The file '/rustc/6d395a1c2946c79966490f5b1e6e619d3a713e6b/library/std/src/thread/local.rs' isn't covered.
[2021-05-14 13:18:47,173 DEBUG] Finished generating per-file code coverage summary.
[2021-05-14 13:18:47,173 DEBUG] Generating file view html index file as: "/out/report/linux/file_view_index.html".
Traceback (most recent call last):
  File "/opt/code_coverage/coverage_utils.py", line 829, in <module>
    sys.exit(Main())
  File "/opt/code_coverage/coverage_utils.py", line 823, in Main
    return _CmdPostProcess(args)
  File "/opt/code_coverage/coverage_utils.py", line 780, in _CmdPostProcess
    processor.PrepareHtmlReport()
  File "/opt/code_coverage/coverage_utils.py", line 578, in PrepareHtmlReport
    self.file_view_path)
  File "/opt/code_coverage/coverage_utils.py", line 450, in GenerateFileViewHtmlIndexFile
    self.GetCoverageHtmlReportPathForFile(file_path),
  File "/opt/code_coverage/coverage_utils.py", line 423, in GetCoverageHtmlReportPathForFile
    self._MapToLocal(file_path)), '"%s" is not a file.' % file_path
AssertionError: "/rustc/6d395a1c2946c79966490f5b1e6e619d3a713e6b/library/std/src/thread/local.rs" is not a file.
```
The failure report: https://bugs.chromium.org/p/oss-fuzz/issues/detail?id=34292
The logs: https://oss-fuzz-build-logs.storage.googleapis.com/log-b5a8362e-201e-47a5-9e19-43b3aa2ab914.txt

I was able to reproduce this locally by running
```
python infra/helper.py build_image oak
python infra/helper.py build_fuzzers --sanitizer coverage oak
python infra/helper.py coverage oak
```

Is it possible to get more details about this error? Is it an error from `llvm-cov`? 
Also, is it possible to disable coverage report generation for project Oak?

--- a/infra/base-images/base-builder/compile
+++ b/infra/base-images/base-builder/compile
@@ -150,17 +150,18 @@ BUILD_CMD="bash -eux $SRC/build.sh"
 
 # We need to preserve source code files for generating a code coverage report.
 # We need exact files that were compiled, so copy both $SRC and $WORK dirs.
-COPY_SOURCES_CMD="cp -rL --parents $SRC $WORK /usr/include /usr/local/include $GOPATH $OSSFUZZ_RUSTPATH $OUT"
+COPY_SOURCES_CMD="cp -rL --parents $SRC $WORK /usr/include /usr/local/include $GOPATH $OSSFUZZ_RUSTPATH /rustc $OUT"
+
+# Copy rust std lib to its path with a hash
+export rustch=`rustc --version --verbose | grep commit-hash | cut -d' ' -f2`
+mkdir -p /rustc/$rustch/
+cp -r /rust/rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/ /rustc/$rustch/
 
 if [ "${BUILD_UID-0}" -ne "0" ]; then
   adduser -u $BUILD_UID --disabled-password --gecos '' builder
   chown -R builder $SRC $OUT $WORK
   su -c "$BUILD_CMD" builder
   if [ "$SANITIZER" = "coverage" ]; then
-    # Copy rust std lib to its path with a hash
-    export rustch=`rustc --version --verbose | grep commit-hash | cut -d' ' -f2`
-    mkdir -p /rust/rustc/$rustch/
-    cp -r /rust/rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/ /rust/rustc/$rustch/
     # Some directories have broken symlinks (e.g. honggfuzz), ignore the errors.
     su -c "$COPY_SOURCES_CMD" builder 2>/dev/null || true
   fi

